docs/changelog.md

```diff
@@ -15,6 +15,9 @@ permalink: /changelog/
 * **feature-prompts**:
   * 🚒 Bug fixed [issue #9229](https://github.com/mozilla-mobile/android-components/issues/9229) - Dismiss SelectLoginPrompt from the current tab when opening a new one ensuring the new one can show it's own. When returning to the previous tab users should focus a login field to see the SelectLoginPrompt again.
 
+* **feature-downloads**:
+  * Browsers could change download notification accent color by providing `Style()` in `AbstractFetchDownloadService` if they want.
```

Comment:
We could improve the message by linking the issue number (https://github.com/mozilla-mobile/android-components/issues/9299) for further references and changing the wording a bit :): 

"**Allow** browsers to change the download notification accent color ...."